app/models/webhook_header.rb

```diff
@@ -0,0 +1,22 @@
+# == Schema Information
+#
+# Table name: webhook_headers
+#
+#  id         :integer          not null, primary key
+#  webhook_id :integer
+#  name       :string(255)
+#  value      :string(255)
```

Comment:
Then with current data structures you can only forward notifications from the registry as they come in. 
   * You cannot forward notifications from portus itself (e.g. someone created a team space.
Solution: Add a column `source` to `webhook` table if this is a desired future feature (I'm neutral). First defined Source is hardcoded and means notification from registry.
   * You cannot filter specific for specific events (e.g. only e specfic repo or specific tag)
Solution: Add a column `filter` of type string(255) to `webhook` table which might be somehow matched against the payload of the event. (I'm neutral too)